### PR TITLE
Charlie creating database

### PR DESCRIPTION
**Pull request by CWeston98**
Added a class that connects to the SpartaGlobal database inside the SQL server we were given

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,7 @@
 boto3
 pandas
-phonenumbers
 pytest
-numpy
\ No newline at end of file
+numpy
+datetime
+setuptools
```
Comment (davidrichardharvey) [inline requirements.txt]: Some of these are base python packages so should not be specified in requirements.

**Comment by davidrichardharvey** [comment]
@CWeston98 This pull request includes .idea files - please remove them!

File: s3_project/classes/create_database.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+import ast
+import json
+
+
+class ProjectDatabase:
+    def __init__(self):
+        self.server = '35.179.41.219'
+        self.database = 'SpartaGlobal'
+        self.username = 'SA'
+        self.password = 'Passw0rd2018'
```
Comment (davidrichardharvey) [inline s3_project/classes/create_database.py]: Uploading passwords to online repositories is **A HUGE NO-NO**

**Review by davidrichardharvey — CHANGES_REQUESTED** [review]

File: s3_project/classes/create_database.py [hunk 3]
```diff
@@ -7,10 +7,10 @@
...
 class ProjectDatabase:
     def __init__(self):
-        self.server = '35.179.41.219'
-        self.database = 'SpartaGlobal'
-        self.username = 'SA'
-        self.password = 'Passw0rd2018'
+        self.server = input('Server:\n')
+        self.database = input('Database:\n')
+        self.username = input('UsernameL\n')
+        self.password = input('Password:\n')
```
Comment (davidrichardharvey) [inline s3_project/classes/create_database.py]: Requiring user input halfway through a potentially automated process is also not great.  You are relying on the user type in an IP address accurately on the first try!

**Review by davidrichardharvey — CHANGES_REQUESTED** [review]

**Review by eviedemetriou — APPROVED** [review]

**Review by davidrichardharvey — APPROVED** [review]

**Review by eviedemetriou — APPROVED** [review]

**Pull request merged**